### ISSUE TITLE
HAKMEM

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1533]
MB; HAKMEM 140

Information from Michael Beeler is that HAKMEM was a plain text file.  AI memo 239 from February 1972 says it's version 140.  Tape backup records show a MB; HAKMEM 141 dated 1972-03-16.

**Comment by larsbrinkhoff** [comment]
There are some characters outside ASCII:

- superscript numbers
- Σ
- √

How were these represented in the text file?  Which printer was used for the hardcopy?

**Comment by Alexey-Slyusar** [comment]
Iverson notation / APL characters set has a lot of non-ASCII characters and it could be displayed and printed at the time. For example IBM Selectric typewriter interchangeable typing element had all the special APL characters on it. 

![default](https://user-images.githubusercontent.com/42380964/52968832-897dbd80-33bf-11e9-9cef-015f7ae79d7d.png)
![default](https://user-images.githubusercontent.com/42380964/52968873-a4503200-33bf-11e9-93e6-b760bc995882.png)

It is possible that the similar mechanism was used at MIT for printing such things like HACKMEM. 
But it is only my guess.

**Comment by larsbrinkhoff** [comment]
Yes, I think so.  There's an AI memo about the Selectric, though it doesn't mention the symbols in HAKMEM.  https://dspace.mit.edu/bitstream/handle/1721.1/6167/AIM-164a.pdf

**Comment by Alexey-Slyusar** [comment]
Lars, thanks for the link!

**Comment by larsbrinkhoff** [comment]
Original file found on ToTS tape 3100151:

MB; HAKMEM 141, dated 1972-03-16
